content/docs/user-guide/data-management/remote-storage.md

```diff
@@ -0,0 +1,114 @@
+# Remote Storage (Optional)
```

Comment:
What do you think about moving "optional" to the intro sentence instead of the title? It makes it long on the sidebar and looks a bit odd to have the parenthetical, especially next to other sections that are also optional but don't have the same:

<img width="299" alt="Screen Shot 2022-10-27 at 9 42 14 AM" src="https://user-images.githubusercontent.com/2308172/198300896-052001d6-4fd5-4d60-906c-56a4bd1b3bd4.png">